src/cli/commands/ping.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const pull = require('pull-stream/pull')
+const drain = require('pull-stream/sinks/drain')
+const pullCatch = require('pull-catch')
+
+const print = require('../utils').print
+
+module.exports = {
+  command: 'ping <peerId>',
+
+  describe: 'Measure the latency of a connection',
```

Comment:
There are several command definitions using `describe` and others using `description`. 

According to [yargs docs](https://github.com/yargs/yargs/blob/master/docs/api.md#completioncmd-description-fn), I think that the correct usage in this case is `description`. In spite of, `describe` working, it is aimed to another use cases, as documented [here](https://github.com/yargs/yargs/blob/master/docs/api.md#describe).

What do you think @JGAntunes and @diasdavid ?

Reply:
fine either way, pick only one though, consistency is key.